MekHQ/src/mekhq/gui/dialog/GMToolsDialog.java

```diff
@@ -331,72 +331,60 @@ private void setUserPreferences() {
         this.setName("dialog");
         preferences.manage(new JWindowPreference(this));
     }
-    
+
     @Override
     public void actionPerformed(ActionEvent event) {
         final String METHOD_NAME = "actionPerformed(ActionEvent)"; //$NON-NLS-1$
 
         if(event.getActionCommand().equals(GM_TOOL_DICE)) {
             performDiceRoll();
         }
-        
+
         if(event.getActionCommand().equals(RAT_ROLLER_TOOL)) {
             lastRolledUnit = performRollRat();
         }
-        
+
         if(event.getActionCommand().equals(RAT_ADDER_TOOL)) {
-            if (null == lastRolledUnit) {
+            if (lastRolledUnit == null) {
                 lastRolledUnit = performRollRat();
             }
 
-            if (null != lastRolledUnit) {
```

Comment:
The second null check is here because performRollRat() can return null.

Reply:
You are correct, this is a dumb mistake on my side. It has been fixed.

Reply:
We all miss simple stuff. That's why we double-check each others' work.